mne/minimum_norm/tests/test_inverse.py

```diff
@@ -344,6 +344,15 @@ def test_apply_inverse_operator():
     assert_true(label_stc.subject == 'sample')
     assert_allclose(stc_label.data, label_stc.data)
 
+    # Test that no errors are raised with loose inverse ops and picking normals
+    noise_cov = read_cov(fname_cov)
+    fwd_orig = make_forward_solution(evoked.info, fname_trans, src_fname,
+                                     fname_bem, eeg=False, mindist=5.0)
```

Comment:
can you avoid recomputing a forward op? just to see if we can keep the test not too long... thanks